tools/IceRpc.Slice.Tools/SliceCCSharpTask.cs

```diff
@@ -95,10 +99,18 @@ protected override void LogEventsFromTextOutput(string singleLine, MessageImport
     {
         if (messageImportance == MessageImportance.Low)
         {
-            // Messages from stdout
-            var jsonDoc = System.Text.Json.JsonDocument.Parse(singleLine);
-            CompilationHash = jsonDoc.RootElement.GetProperty("hash").GetString();
-            return;
+            try
+            {
+                // Messages from stdout
+                var jsonDoc = System.Text.Json.JsonDocument.Parse(singleLine);
+                CompilationHash = jsonDoc.RootElement.GetProperty("hash").GetString();
+                ContainsSlice1 = jsonDoc.RootElement.GetProperty("contains_slice1").GetString();
+                return;
+            }
+            catch (System.Text.Json.JsonException)
+            {
+                Log.LogError(singleLine);
```

Comment:
Should we print part of the `JsonException` here too? Just to provide some details about what actually failed for when we're trying to debug this 5 years from now : vP